### ISSUE TITLE
Temperature Parameter in a Large Language Model (LLM)

### ISSUE THREAD

**Issue by hammad93** [hammad93/ai-seed#1]
The temperature parameter is often referred to as the level of "randomness" during the inference for a LLM. Knowing that this repository is focused on the application of TRNG's to AI and LLM's, this issue is to track analysis, ideas, and suggestions. For example, the temperature is related to the softmax function but there is also the sampling step, where a TRNG can be incorporated.

**Comment by hammad93** [comment]
The temperature parameter is algorithmically a factor in the sampling step of the next token prediction. It depends on the exact loss function and how we choose which tokens will be included in the sample, but this sampling step depends on random number generation.
